integrations/aws/overrides.py

```diff
@@ -0,0 +1,16 @@
+from port_ocean.core.handlers.port_app_config.models import (
+    ResourceConfig,
+    PortAppConfig,
+    Selector,
+)
+from pydantic import BaseModel, Field
+
+class AWSSelector(Selector):
+    resource_kinds: list[str] = Field(alias="resourceKinds", default=[], min_items=1)
```

Comment:
resource_kinds are only supported for the `cloudResource` kind, you need to restrict so it allowed only for that kind

Reply:
But what if a user would like to create his own custom kind that utilises this?